Userland/Libraries/LibCore/DateTime.cpp

```diff
@@ -538,8 +538,8 @@ Optional<DateTime> DateTime::parse(StringView format, StringView string)
                 hours = hours / 100;
             }
 
-            tm.tm_hour += sign * hours;
-            tm.tm_min += sign * minutes;
+            tm.tm_hour -= sign * hours;
+            tm.tm_min -= sign * minutes;
```

Comment:
Can we test this?

Reply:
Yes, since it breaks in JS. It might have been fixed by Jelle’s changes earlier, so after some reading I think I will just drop this PR for now.